main.js

```diff
@@ -0,0 +1,298 @@
+let startButton2 = document.getElementById('start2')
+let startButton1 = document.getElementById('start1')
+
+startButton2.addEventListener('click', startGame2)
+startButton1.addEventListener('click', startGame1)
+
+// ------------------------------------------------------------------------------------
+// player objects and global variables
+// player class
+
+class Player {
+  constructor(name, symbol) {
+    this.name = name
+    this.symbol = symbol
+  }
+}
+
+// player objects
+
+let playerX = new Player("", "X")
+let playerO = new Player("", "O")
+let currentPlayer
+
+// varibles for getting inputted player names from text input field
+
+let xName = document.getElementById("playerX-name")
+let oName = document.getElementById("playerO-name")
+
+// variables to retrieve cell info
+
+let c0 = document.getElementById('cell-0')
+let c1 = document.getElementById('cell-1')
+let c2 = document.getElementById('cell-2')
+let c3 = document.getElementById('cell-3')
+let c4 = document.getElementById('cell-4')
+let c5 = document.getElementById('cell-5')
+let c6 = document.getElementById('cell-6')
+let c7 = document.getElementById('cell-7')
+let c8 = document.getElementById('cell-8')
+
+// other global variables
+
+let cells = document.querySelectorAll("[id^='cell']")
+let playerStatus = document.getElementById("status-bar")
+
+// -----------------------------------------------------------------------------------------------
+// functions
+// function to check whether any winning combination has been achieved during play
+
+function gameWin() {
+  let index = 0
+
+  // loop to iterate through winning array combinations
+
+  while (index < 8) {
+
+    // winning array combinations
+
+    let winningArrays = [
+
+      [c0.innerHTML, c1.innerHTML, c2.innerHTML],
+      [c0.innerHTML, c3.innerHTML, c6.innerHTML],
+      [c0.innerHTML, c4.innerHTML, c8.innerHTML],
+      [c3.innerHTML, c4.innerHTML, c5.innerHTML],
+      [c6.innerHTML, c7.innerHTML, c8.innerHTML],
+      [c1.innerHTML, c4.innerHTML, c7.innerHTML],
+      [c2.innerHTML, c5.innerHTML, c8.innerHTML],
+      [c2.innerHTML, c4.innerHTML, c6.innerHTML]
+
+    ]
+    // function to filter arrays for X's and O's
+
+    checkedMoves = winningArrays[index].filter(function (value) {
+      return value === currentPlayer.symbol
+    })
+    index++
+
+    // check that returns the first array to contain 3 X's or O's
+
+    if (checkedMoves.length === 3) {
+      console.log(index) // I'm not sure how to retrieve (return) this index, but I think I'll need it in order to know which combination was reached so that I can draw a line through the appropriate cells or change their background color.
+      return checkedMoves
+    }
+  }
+}
+
+// function to delay game (i.e. page) reset
+
+function delayEnding() {
+  window.setTimeout(endGame, 4 * 1000)
+}
+
+// function to reset board (i.e. page)
+
+function endGame() {
+  document.location.reload();
+  clearInterval(interval);
```

Comment:
Here you can just reload the page. The clearInterval function is used to clear an interval set with the setInterval function, which you can actually use to create your timer